contracts/KeysManager.sol

```diff
@@ -491,6 +494,10 @@ contract KeysManager is EternalStorage, IKeysManager {
         _setIsPayoutActive(false, _miningKey);
     }
 
+    function _getValidatorMetadata() private view returns(address) {
```

Comment:
I recommend to return `IValidatorMetadata`, not `address` as a more type-safe approach.

Reply:
@Eenae yes, thanks for your proposal. I'll review all the contracts to apply the same approach for the similar private functions.